src/components/AppLayout.jsx

```diff
@@ -28,11 +33,36 @@ export function AppLayout({ children }) {
         <div className="flex-none">
           <div className="flex flex-row gap-4 items-center">
             <Image alt="bell" className="size-7" src="/bell.svg" />
-            <Image alt="profile" className="size-10" src="/profile.svg" />
-            <div className="flex flex-col text-black">
-              <p>เจ้าหน้าที่ส่วนกลาง</p>
-              <p className="text-sm font-light">เจ้าหน้าที่ส่วนกลาง</p>
-            </div>
+            <Menu
+              shadow="md"
+              trigger="click-hover"
+              width={200}
+              transitionProps={{ transition: "scale-y" }}
+            >
+              <Menu.Target>
+                <div className="row items-center gap-4 hover:cursor-pointer">
+                  <Image alt="profile" className="size-10" src="/profile.svg" />
+                  <div className="flex flex-col text-black">
+                    <p className="text-sm">เจ้าหน้าที่ส่วนกลาง</p>
+                    <p className="text-xs font-light">เจ้าหน้าที่ส่วนกลาง</p>
+                  </div>
+                  <IconChevronDown size={16} />
+                </div>
+              </Menu.Target>
+
+              <Menu.Dropdown>
+                <Menu.Item leftSection={<IconSettings size={14} />}>
+                  ตั้งค่า
```

Comment:
Not sure that should we show this one? 🤔

Reply:
I write this mock cause of มันดูมีอะไรดี 55555, Anyway if team (includes p'sorn) think it's not necessary We can remove it kub.